addons/hr_payroll_account/tests/test_hr_payroll_account.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+
+import time
+from datetime import datetime, timedelta
+from dateutil import relativedelta
+
+from odoo import fields, tools
+from odoo.modules.module import get_module_resource
+from odoo.tests import common
+
+
+class TestHrPayrollAccount(common.TransactionCase):
+
+    def _load(self, module, *args):
```

Comment:
Instead of loading the xml, which is strange, you can inherit from the account test class which already defined the data in the setUp() method.
``` python
from openerp.addons.account.tests.account_test_classes import AccountingTestCase


class TestHrPayrollAccount(AccountingTestCase):
```